src/data/spleenhit.txt

```diff
@@ -29,8 +29,8 @@ ancient protein powder	1	1	good	0	12-14	0	0
 antimatter wad	2	3	awesome	5-7	-20-30	-20-30	-20-30
 antique bottle of cough syrup	1	6	awesome	0	0	18-20	0	10 The Q Is Talking To You (+10% mys)
 armored prawn	1	1	EPIC	0	30-50	30-50	30-50	50 Rushtacean' (+50 Init)
-[5140]astral energy drink	8	1	awesome	29-35	0	0	0
 [10883]astral energy drink	5	1		0	0	0	0	Lucky!
+[5140]astral energy drink	8	1	awesome	29-35	0	0	0
```

Comment:
Any other change besides applying consistent sorting?

Reply:
nope

Reply:
OK.  Couldn't figure out what had changed or why the line was moved.